tests/test_cases/mknod.c

```diff
@@ -0,0 +1,35 @@
+#undef _GNU_SOURCE
+#define _GNU_SOURCE
+
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+#define DEVICE_PATH "/dev/test" // Defining the path for the device to be created
```

Comment:
`/dev/test` doesn't exist in lindfs so maybe make use of one of the following:

- /dev/null 
- /dev/urandom 
- /dev/random 
- /dev/zero

Or just create a new file in general and use that.